### PR TITLE
Created methods to set coast/brake mode and open loop ramp rate settings

### PR DESCRIPTION
**Pull request by cgjyang**

### CODE REVIEW

**Comment by jmullins16** [comment]
Merged with Naina's branch to remove conflicts. Looks good and makes it obvious the modes are only active then DriveTelop command is running. Need another approval before merge.

**Review by ngupta-22 — APPROVED** [review]
Nice! Having init and end methods makes the code look cleaner

**Pull request merged**
